contracts/Penalizer.sol

```diff
@@ -84,7 +84,7 @@ contract Penalizer {
         if (decodedTx.to == address(hub)) {
             bytes4 selector = GsnUtils.getMethodSig(decodedTx.data);
             require(
-                selector != hub.relayCall.selector,
+                selector != IRelayHub.relayCall.selector,
```

Comment:
excelent. I was trying to look for a way to reference a method without having an instance! is it new in sol6?

Reply:
Yes, this is solc v6

Reply:
Yes I tried but solc 5 didn't let me when writing it...